.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,25 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+ Fuzzing:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Build Fuzzers
+     id: build
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+     with:
+       oss-fuzz-project-name: 'quick-xml'
+       language: c++
+   - name: Run Fuzzers
+     uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
+     with:
+       oss-fuzz-project-name: 'quick-xml'
+       language: c++
+       fuzz-seconds: 600
+   - name: Upload Crash
+     uses: actions/upload-artifact@v3
+     if: failure() && steps.build.outcome == 'success'
+     with:
+       name: artifacts
+       path: ./out/artifacts
```

Comment:
That name / path has some concrete meaning for the job? Maybe better to name them `crashes` if the concrete name do not matter?

Reply:
Yeah it has a specific meaning in the context of fuzzing. Artifacts encompass more than just crashes. e.g. crashes, out-of-memory errors, timeouts (e.g. triggered by an infinite loop) etc. I think even if we could change the name of this directory (which oss-fuzz won't like), it wouldn't make sense to as some of the outputs here aren't necessarily just crashes. There is some documentation for artifacts [here](https://www.llvm.org/docs/LibFuzzer.html).